### PR TITLE
fix: strings-need-to-be-json-objects

### PR DESCRIPTION
**Pull request by jahead**

### CODE REVIEW

File: src/android/StorageMigrator.java [hunk 1]
```diff
@@ -122,18 +122,21 @@ private void writeToNativeStorage(String key, String value, ValueType type, Shar
...
             case STRING:
             case OBJECT:
-                editor.putString(key, value);
+                try {
+                    // Mimicking `setItem` that calls `JSON.stringify(value)`
+                    JSONObject object = new JSONObject(value);
+                    editor.putString(key, object.toString());
+                } catch (JSONException e) {
+                    Log.e(TAG, e.getMessage());
+                }
```
Comment (franco-octavianus) [inline src/android/StorageMigrator.java:136]: does this work without imports?

**Review by franco-octavianus — APPROVED** [review]

File: src/android/StorageMigrator.java (hunk 1 above)
Comment (jahead) [reply src/android/StorageMigrator.java:136]: no.... oops copy paste errors

Comment (aaronadrid-isv) [reply src/android/StorageMigrator.java:136]: Is this fixed?

Comment (jahead) [reply src/android/StorageMigrator.java:136]: is now :D

**Pull request merged**